Updated homepage and added the project status link

natashapl opened the pull request:
### This pull request includes the following update:

- Updated The Notify.gov brow on the homepage
- Added the project status link to the top nav and changed the style to match @jonathanbobel 's design.

[PR Preview](https://federalist-37831b52-7633-429a-9a79-14298034dd54.sites.pages.cloud.gov/preview/gsa-tts/10x-website/add-project-status-link-to-menu/)

jonathanbobel commented on _includes/components/header.html:
```diff
@@ -55,7 +55,109 @@
+    <button type="button" class="usa-menu-btn">Menu</button>
+  </div>
+  <nav aria-label="Primary navigation" class="usa-nav">
+    <div class="usa-nav__inner">
```
I was messing around with the z-index here and bumping it up allowed for the tabbed focus state to be visible over the gradient background below. That might not be the best solution, so I'd experiment in the code.

jonathanbobel replied:
(specifically "usa-nav__inner")

natashapl replied:
Hi @jonathanbobel . The branch has been updated. Please review.

natashapl commented:
Hi @iamjolly . Hope all is well. In this task, we've updated the main top navigation of the 10x website and added a "Project Status" link. When you have a moment, could you please perform some additional accessibility testing and let us know if we overlooked anything? Thanks so much!

[Click here to access the branch.](https://federalist-37831b52-7633-429a-9a79-14298034dd54.sites.pages.cloud.gov/preview/gsa-tts/10x-website/add-project-status-link-to-menu/)

iamjolly commented:
@natashapl - Things check out fine in general on the new link, with a couple of minor comments/suggestions:

1. The Github icon is cool, but the accessible name for the link is simply "Project Status". I think as-is, it is OK, but would be even better to be "Project Status on Github" as the accessible name for the link. For this, you may want to use visually-hidden text in a `span` within the `a` to add to the link text. It's optional, but seems like it could be a nice touch.
2. Focus outline is great for this link and is fine as-is. I really like the `:hover` color change from white to light blue and would recommend that also be applied to the `:focus` in the CSS, along with the focus outline. Again, not a big deal if it's not implemented, just a nice touch visually.

natashapl commented:
Hi @iamjolly . Thanks for you suggestions. They've both been implemented. Please review when you get a chance. By the way, I added you as a reviewer. Thanks.

iamjolly reviewed (approved):
these look good to me - thanks for working through this!

jonadecker commented:
This looks really good @natashapl! Thanks for your help on this @iamjolly

jonathanbobel reviewed (commented):
LGTM
